subprojects/gst-plugins-good/gst/rtp/gstrtph266pay.c

```diff
@@ -653,34 +645,31 @@ _extract_fu (GstRtpH266Pay * rtph266pay, const Nalu * nalu,
 
   fu_pyl_buf = gst_buffer_copy_region (nalu->rbsp_buf, GST_BUFFER_COPY_MEMORY,
       offset, size);
-  if (!fu_pyl_buf)
-    return NULL;
+  g_assert (fu_pyl_buf);
```

Comment:
nitpick: `g_assert ((fu_pyl_buf = gst_buffer_copy_region (nalu->rbsp_buf, GST_BUFFER_COPY_MEMORY,
      offset, size)));`

Similar in other similar cases.

Reply:
:point_right: https://github.com/fluendo/gstreamer/pull/37#discussion_r1847782501